### PR TITLE
Update card styles for Infra-Chaos tag

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: oumkale <oum.kale@mayadata.io>

## Proposed changes

Update card styles for Infra-Chaos tag
## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/components/WorkflowCard/styles.ts [hunk 1]
```diff
@@ -191,7 +191,7 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: 'row',
     alignItems: 'center',
     padding: theme.spacing(0.275, 0.75, 0.275, 0.75),
-    backgroundColor: theme.palette.secondary.light,
+    backgroundColor: theme.palette.secondary.dark,
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/WorkflowCard/styles.ts:194]: It is yellow as a warning in my hub and chaos hub @oumkale

Comment (oumkale) [reply litmus-portal/frontend/src/components/WorkflowCard/styles.ts:194]: Do we have yellow theme?

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/WorkflowCard/styles.ts:194]: Yes @oumkale

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/WorkflowCard/styles.ts:194]: Check ‘theme.palette.warning.main’

Comment (oumkale) [reply litmus-portal/frontend/src/components/WorkflowCard/styles.ts:194]: It's not exactly same. In hub it is #F6B92B and using theme #dba017. cc : @arkajyotiMukherjee

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/components/WorkflowCard/styles.ts:194]: You may use any yellow theme colour, litmus ui is not used in chaos hub currently @oumkale

**Review by ishangupta-ds — APPROVED** [review]
LGTM

**Review by arkajyotiMukherjee — APPROVED** [review]

**Pull request merged**